developers_chamber/ecs_utils.py

```diff
@@ -449,6 +450,29 @@ def get_tasks_for_service(cluster, service, region):
     return tasks_arns
 
 
+def stop_services_and_wait_for_tasks_to_stop(cluster, services, timeout, region):
+    ecs_client = _get_ecs_client(region)
+
+    THREAD_MAX = 32
+    number_of_services = len(services)
+
+    tasks = []
+    pool = ThreadPool(number_of_services if number_of_services < THREAD_MAX else THREAD_MAX)
+
+    for service in services:
+        service_tasks = get_tasks_for_service(cluster=cluster, service=service, region=region)
+        if not service_tasks:
```

Comment:
```
if service_tasks:
    stop_service(cluster=cluster, service=service, region=region)
else:
     LOGGER.info("No active tasks found in service '{}'".format(service))
````

Reply:
continue je fuj

Reply:
ok netušil jsem že je to fuj :) předěláno